src/mocks/handlers.ts

```diff
@@ -1000,18 +1001,63 @@ export const handlers = [
 
     return HttpResponse.json(
       {
-        id: '1',
         ...body,
-        title: body.title ?? null,
-        description: body.description ?? null,
         upload_info: upload_info,
-        modified_time: '2024-01-02T13:10:10.000+00:00',
-        created_time: '2024-01-01T12:00:00.000+00:00',
       },
       { status: 201 }
     );
   }),
 
+  http.get('/attachments', ({ request }) => {
+    const url = new URL(request.url);
+    const attachmentParams = url.searchParams;
+    const entityId = attachmentParams.get('entity_id');
+
+    if (entityId === '90') {
+      return HttpResponse.json([], { status: 200 });
+    } else if (entityId === '3') {
+      return HttpResponse.json(
+        [
+          {
+            ...AttachmentsJSON[0],
+            entity_id: entityId,
+            ...(entityId === '3' && { file_name: 'test '}),
+          },
+        ],
+        { status: 200 }
+      );
+    }
+
+    const generateAttachments = () => {
+      return Array.from({ length: 20 }, (_, index) => {
+        const id = index + 1;
+        let attachment;
+
+        if (Number(id) % 4 === 0) {
+          attachment = AttachmentsJSON[0];
+        } else if (Number(id) % 4 === 1) {
+          attachment = AttachmentsJSON[1];
+        } else if (Number(id) % 4 === 2) {
+          attachment = {
+            ...AttachmentsJSON[2],
+            ...(id === 3 && {
+              file_name: 'test',
+              description: undefined,
+            }),
+          };
+        } else if (Number(id) % 4 === 3) {
+          attachment = AttachmentsJSON[3];
+        }
+        return {
+          ...attachment,
+          id: String(id),
+        };
+      });
+    };
+
+    return HttpResponse.json(generateAttachments(), { status: 200 });
+  }),
+
```

Comment:
```suggestion
  http.get('/attachments', ({ request }) => {
    const url = new URL(request.url);
    const attachmentParams = url.searchParams;
    const entityId = attachmentParams.get('entity_id');

    const generateAttachments = () => {
      return Array.from({ length: 20 }, (_, index) => {
        const id = index + 1;
        const attachment = { ...AttachmentsJSON[id % 4] };

        return {
          ...attachment,
          id: String(id), // Ensure the id is a string
          entity_id: entityId,
        };
      });
    };

    return HttpResponse.json(generateAttachments(), { status: 200 });
  }),
```

This handler can be implemented like this. The description being undefined was a specific condition needed for the image testing